administrator/language/de-DE/plg_system_actionlogs.ini

```
@@ -11,6 +11,7 @@ PLG_SYSTEM_ACTIONLOGS_INFO_LABEL="Information"
 PLG_SYSTEM_ACTIONLOGS_JOOMLA_ACTIONLOG_DISABLED="Benutzeraktivitäten – Joomla"
 PLG_SYSTEM_ACTIONLOGS_JOOMLA_ACTIONLOG_DISABLED_REDIRECT="Das %s Plugin ist deaktiviert."
 PLG_SYSTEM_ACTIONLOGS_NOTIFICATIONS="Benachrichtung über Benutzeraktivitäten senden"
+PLG_SYSTEM_ACTIONLOGS_NOTIFICATIONS_EXCLUDE="Selbst ausschließen"
```

```suggestion
PLG_SYSTEM_ACTIONLOGS_NOTIFICATIONS_EXCLUDE="Sich selbst ausschließen"
```
Übersetzung ist Ok, aber könnte unverständlich sein.